guides/web-apps/acceptance-testing-with-selenium/README.md

```diff
@@ -58,50 +58,20 @@ Run the following commands in the console to install the required dependencies:
 ```
 Install-Package Selenium.WebDriver
 Install-Package Selenium.Support
-Install-Package NUnit -Version 2.6.4
-Install-Package NUnitTestAdapter
-Install-Package NUnit.Runners -Version 3.2.0
+Install-Package NUnit -Version 3.*
+Install-Package NUnit.ConsoleRunner -Version 3.*
+Install-Package NUnit3TestAdapter
 ```
 
-## Record your first test
-
-"Record" the test? Yes, you read this right! While you can also write the test code manually using Selenium's domain-specific language, it is way more effective to record it directly from a Firefox extension.
-
-Make sure you have Selenium IDE extension installed in Firefox. The instructions are here: [http://www.seleniumhq.org/projects/ide/](http://www.seleniumhq.org/projects/ide/).
-
-Make sure that the project that you want to test is already running. For me it means, that Launcher is started and accessible at `http://localhost:8080/launcher`.
-
-To record the first test, go to Firefox, and click on the Selenium IDE icon.
-
-![record first test location](/assets/2016-04-01-13_24_19-Mozilla-Firefox.png)
-
-When the Selenium IDE opens, it automatically starts recording.
-
-![selenium IDE](/assets/2016-04-01-13_25_24-Mozilla-Firefox.png)
-
-Now, you need to focus your main Firefox window again. Type in the URL of your app, press ENTER and wait until the page loads.
-
-When the page is loaded, right click anywhere on the page. In the context menu, select <kbd>Show All Available Commands</kbd> → <kbd>waitForTitle</kbd>. This records an action of waiting for a page title to appear in the window title bar. This will make a good first test to see if your app loads correctly.
-
-Now go back to Selenium IDE and press the red circle to stop recording. The recorded output should look like this:
-
-![start recording](/assets/2016-04-01-13_29_20-Launcher.png)
-
-Now, you can save this recording as a `.cs` file. In Selenium IDE, click <kbd>File</kbd> → <kbd>Export Test Case As…</kbd> → <kbd>C# / Nunit / WebDriver</kbd>.
-
-In the file dialog, go to your test project directory and save the file as `PageLoads.cs`
-
 ## Run your first test
 
-Now, import the recorded file to your project in Visual Studio. Right click on the project directory, select <kbd>Add</kbd> → <kbd>Existing item…</kbd> and pick the file `PageLoads.cs`. Open the file.
-
-Still in Visual Studio, open the Test Explorer window pane, by clicking on <kbd>Test</kbd> → <kbd>Windows</kbd> → <kbd>Test Explorer</kbd>.
+Clone KitchenSink project from StarcounterSamples repository.
```

Comment:
Should be: Clone [KitchenSink](https://github.com/StarcounterApps/KitchenSink) repo from the StarcounterApps organisation on GitHub.